admin_guide/scheduler.adoc

```diff
@@ -287,6 +288,13 @@ of applications by allowing administrators to ensure that peer pods do not end
 up being too geographically separated. If no node is available within the same
 affinity group to host the pod, then the pod will not get scheduled.
 
+If you need greater control over where pods are scheduled, see 
```

Comment:
@mburke5678 Can you revisit this one? Please also check all instances of the words `will` & `in order to` and use it only where it is necessary. Rest is all good. This was a big one indeed. :clap: